### DISCUSSION TITLE
Uptime...

### DISCUSSION THREAD

**Discussion by jornobe** [theyosh/TerrariumPI#741] [Show and tell]
Getting ready to upgrade to the latest release. Too bad it will require the system to restart :)

![image](https://user-images.githubusercontent.com/45115923/195629758-245e5eea-74b6-40a6-9275-c775d5d99d7e.png)

Very happy with how stable it all runs. This one is maintaining proper environmental values for one of my larger ant colonies.

**Comment by theyosh** [comment]
Nice... I need some more days I see :)

![image](https://user-images.githubusercontent.com/8080177/195643024-ffc43c86-c78b-4ae6-809a-9f853d49f980.png)

**Comment by theyosh** [comment]
A restart of the Pi is not needed for an upgrade. A service restart is enough, that is why I have now almost 5 months :P
But I have now for 5 years of data, which means my hardware is now running for at least 5 years without breakdowns!!! 

![image](https://user-images.githubusercontent.com/8080177/205387346-af26089c-2c9f-4dcb-9a82-5a6e61037223.png)

**Comment by theyosh** [comment]
Highest uptime so far. 6 months and a few days in July 2023!!
![image](https://github.com/theyosh/TerrariumPI/assets/8080177/daab3104-fcb3-478e-8562-17bcdbdc2c04)
